src/main/java/wolox/training/models/User.java

```diff
@@ -64,6 +65,10 @@
     @Getter
     private String username;
 
+    @Getter
+    @Setter
+    private String password;
```

Comment:
Add a PUT endpoint into UserController that allows updating only the password.

The password must always be modified using a resource other than updating a User